docs/index.md

```diff
@@ -6,10 +6,30 @@ A new kind of BIOS firmware that will solve the problems of ownership, security
 and compatibility allowing you create secure, efficient images that are fully
 customizable to your tastes and needs.
 
+Dasharo is a set of productized services, Open Core, and SaaS products which
+help to provide scalable, modular, easy to combine Open Source BIOS, UEFI, and
+Firmware solutions. It offers the components that are needed to develop and
+maintain a high quality, and modular firmware, for the stability and security
+of your platform.
+
 [Dasharo homepage](https://dasharo.com/)
 
-## Variants
+## Stock Keeping Units (SKUs)
+
+Also called variants or flavours. In essence Dasharo SKUs are set of Open
+Source Firmware releases produced for given market segment with awareness of
+features that given segment requires. Each market segment includes two types of
+platforms Reference Platform (RP) and Hardware Compatibility List Platforms
+(HCLP).
+
+What SKUs we support:
 
 * [Dasharo Workstation](variants/workstation.md)
+* [Dasharo Firewall](variants/firewall.md)
 * [Dasharo Secure Firewall](variants/secure-firewall.md)
-* [Dasharo Trustworthy Server](variants/trustworthy-server.md)
+* [Dasharo Trustworthy Server CE](variants/trustworthy-server.md)
```

Comment:
Dasharo Trustworthy Server **CE** = Community Edition?

Reply:
@pietrushnic Maybe CE isn't necessary at all if you don't have a regular "Dasharo Trustworthy Server" category now, but considering CE doesn't occupy much space it could stay _(and no need to expand, to reduce the chance of reader's "TLDR")_

Reply:
@mikebdp2 I will leave it - we always can change in the future.